test/disabled/basic_single_server_auth.test.js

```diff
@@ -44,7 +44,7 @@ describe('Basic single server auth tests', function() {
         expect(r).to.exist;
 
         const server = config.newTopology(this.configuration.host, this.configuration.port, {
-          bson: new Bson()
+          bson: BSON
```

Comment:
let's consider moving this to `newTopology` itself, so we don't have to specify it so often in all of these tests. The `bson` property of `options` should only be used when we want to explicitly override.